ts_scripts/install_dependencies.py

```diff
@@ -38,6 +38,13 @@ def install_torch_packages(self, cuda_version):
                 os.system(
                     f"{sys.executable} -m pip install -U -r requirements/torch_{cuda_version}_{platform.system().lower()}.txt"
                 )
+        elif args.neuronx:
```

Comment:
why do you have 2 files for neuron dependencies and can't we just roll them into the same `if` condition if they're both needed?